fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRunHistory.java

```diff
@@ -66,9 +71,23 @@ public List<TaskRunStatus> getAllHistory() {
 
     public void forceGC() {
         List<TaskRunStatus> allHistory = getAllHistory();
+        int beforeSize = allHistory.size();
```

Comment:
will it run concurrently?

Reply:
`forceGC` should be thread-safe. since `allHistory ` is copied .